backside/docker-compose.yml

```diff
@@ -1,5 +1,17 @@
-version: "3.9"
+version: '3.7'
+
 services:
+  postgres:
+    image: postgres:latest
+    restart: always
+    environment:
+      - POSTGRES_USER=root
+      - POSTGRES_PASSWORD=root
+    ports:
+      - "5432:5432"
```

Comment:
After merging my [pull request (please review)](https://github.com/sk8mate/sk8/pull/46) we would need some changes here but I'll handle it.

Reply:
what changes?